src/api/nft-namespace.ts

```diff
@@ -473,6 +477,33 @@ export class NftNamespace {
     return getSpamContracts(this.config);
   }
 
+  /**
+   * Returns whether a contract is marked as spam or not by Alchemy. For more
+   * information on how we classify spam, go to our NFT API FAQ at
+   * https://docs.alchemy.com/alchemy/enhanced-apis/nft-api/nft-api-faq#nft-spam-classification.
+   *
+   * @param contractAddress - The contract address to check.
+   * @beta
+   */
+  reportSpam(contractAddress: string): Promise<ReportSpamResponse> {
```

Comment:
Since the return string of a successful response is always just a message indicating the NFT was reported successfully, can we return `Promise<void>` in this method and just not return the underlying string?

We can also delete `ReportSpamResponse`.